src/Views/Form/Templates/Classic/resources/views/header.php

```diff
@@ -2,42 +2,60 @@
 /**
  * @var string $title
  * @var string $description
- * @var bool   $isSecureBadgeEnabled
- * @var bool   $secureBadgeContent
+ * @var bool $isSecureBadgeEnabled
+ * @var bool $secureBadgeContent
+ * @var bool $hasGoal
+ * @var array $goalStats
  */
+
 ?>
 <div class="give-form-header">
     <div class="give-form-header-top-wrap">
         <h1 class="give-form-title"><?= $title ?></h1>
         <p class="give-form-description"><?= $description ?></p>
-        <?php if ($isSecureBadgeEnabled) : ?>
-        <aside class="give-form-secure-badge">
-            <svg class="give-form-secure-icon">
-                <use href="#give-icon-lock" />
-            </svg>
-            <?= $secureBadgeContent ?>
-        </aside>
-        <?php endif; ?>
+        <?php
+        if ($isSecureBadgeEnabled) : ?>
```

Comment:
I know PSR-12 is doing this and this comment is unrelated to this PR, but I think we should not apply PSR-12 to files that are substantially HTML. You can just leave this though.

Reply:
@JasonTheAdams Do you think we should use `.html.php` and then exclude that extension from being formatted to PSR-12, so stuff like this doesn’t happen in HTML?

Reply:
Would we use prettier instead, then?

Reply:
I guess, ya. I haven’t really seen how Prettier formats HTML.

Reply:
This may kill you to hear, but I'm less interested in visual perfection and more interested in consistency. So I don't want to exclude something without a working and reliable alternative.

Reply:
I just tried Prettier with HTML and I think it's good.

My thing is that the formatter should serve the primary language that it's formatting and when most of the file is HTML it just makes sense to me to format it like HTML.

Reply:
Oh, man. Code formatting is hard.

Reply:
Cause code formatting is really to serve the reader, right?

Reply:
I'm also fine with just leaving this. We'll probably switch to React in GiveWP 3.

Reply:
Yeah, let's just move along for now. You're right, this will likely become React.

Formatting, to me, falls into a 90/10 trap. That is, most standards will take code 90% of the way to making it entirely readable. In fact, it's probably more. The last 10% (should curly brackets be on the same line!?!?) is just personal preference and highly subjective. So long as code is sensibly readable (90%), I don't care. At that point I just want consistency so our commits aren't ridiculous and more difficult to review.

Reply:
You know what I don’t know if this is actually formatted with PSR-12 though. I just ran the `phpcbf --standard=PSR12` on it and it doesn’t seem to have any issues with keeping the `if` and `endif` on the same lines.

Reply:
Resolved by https://github.com/impress-org/givewp/commit/f9a6954e544f6a04f2bf0197e950c6cc829cfb1e

PSR-12 actually doesn’t create issues formatting files with lots of HTML. We can carry on.